crt_portal/static/sass/custom/landing.scss

```diff
@@ -18,8 +18,12 @@ $arrow-height: 24px;
   }
 
   h2 {
-    margin-top: 0;
+    margin-top: 1.5rem;
     margin-bottom: 0;
+
+    @include at-media-max(mobile-lg) {
+      margin-top: 0;
+    }
```

Comment:
Adding back in 1.5rem but making exclusion for mobile should solve the desktop spacing and allow mobile to h2 to collapse where needed (I found out that there are a couple of places that need h2 margin-top 0). Thanks @hbillings.

Reply:
fantastic, thank you! However, this re-introduces [the spacing bug we were trying to squash in the last round](https://github.com/usdoj-crt/crt-portal/pull/589#pullrequestreview-462571347) because we still need to add the piece I mentioned in my last review:
> Somewhere in this file, there's a `crt-landing--reporting_column` class. You want to write a rule that sets `.crt-landing--reporting_column h2` to a top margin of 0

Once we've got that in, this should be golden 🤞